md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/LineageAPI.java

```diff
@@ -105,7 +105,7 @@ RestWrapper getByBatch(@PathVariable("batchId") String batchId, Principal princi
             if (hasBatch) {
                 SimpleDateFormat format = new SimpleDateFormat("MM-dd-yyyy HH:mm:ss");
 
-                StringBuffer InstanceExecLabel = new StringBuffer("<<TABLE ALIGN=\"LEFT\" CELLSPACING=\"4\" CELLPADDING=\"0\" BORDER=\"0\" WIDTH=\"100%\">" +
+                StringBuilder instanceExecLabel = new StringBuilder("<<TABLE ALIGN=\"LEFT\" CELLSPACING=\"4\" CELLPADDING=\"0\" BORDER=\"0\" WIDTH=\"100%\">" +
                         "<TR><TD><FONT POINT-SIZE=\"9\">Process Exec#" + "</FONT></TD></TR>" +
                         "<TR><TD><FONT POINT-SIZE=\"12\">" + instanceExecId + "</FONT></TD></TR>" +
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "<TR><TD><FONT POINT-SIZE=\"12\">" 6 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1192)